tests/eval/good/builtin-isqrt.scilexp

```diff
@@ -0,0 +1,19 @@
+let zero = Uint32 0 in
+let one = Uint32 1 in
+let two = Uint64 2 in
+let three = Uint32 3 in
+let four = Uint128 4 in
+let six = Uint32 6 in
+let ten = Uint32 10 in
+let thirty_six = Uint256 36 in
+
```

Comment:
Could you add one more example with the largest number for some uint type? Perhaps `Uint128`?